transitions/pixelize.glsl

```diff
@@ -2,8 +2,8 @@
 // License: MIT
 // forked from https://gist.github.com/benraziel/c528607361d90a072e98
 
-uniform ivec2 squaresMin/* = ivec2(20) */; // minimum number of squares (when the effect is at its higher level)
-uniform int steps /* = 50 */; // zero disable the stepping
+uniform ivec2 squaresMin; // = ivec2(20) // minimum number of squares (when the effect is at its higher level)
```

Comment:
uniform 'steps' default value '= 50 // zero disable the stepping' does not parse as GLSL code: unexpected EOF at line 1